ulp-observability-front/src/main/front/src/app/components/ulp-observability-dashboard/ulp-observability-dashboard.component.ts

```diff
@@ -32,6 +32,7 @@ export class UlpObservabilityDashboardComponent implements OnInit{
   private acceptedPostfixs = ['total', 'avg', 'avg_every_periods', 'max', 'max_every_periods', 'throughput', 'throughput_every_periods', 'threads', 'threads_every_periods', 'pct'];
   private updateFrequencyS = 60;  
   
+  numberTopErrors = 10;
```

Comment:
is this needed  ?
Shouldn't it be aligned with server default value 5 ?

Reply:
oh yes i should change it to 5. I guess i will create a file to store hard coded constants used in front. Because i see that the same constant may be declared several times in different files.